space-concordia-robotics/robotics-prototype
# Removed Useless C++ Files

# Assignee Section

## Description
I removed a bunch of useless C++ files such as `ideas.h` or `notes.h` as well as a bunch of test projects. This makes the repo much cleaner.

### Steps for Testing
Ensure that you have the arduino IDE setup from the wiki and can build with cmake.
1. In rover/, do `mkdir build`
2. `cd build`
3. `cmake ..`
4. `make`

Everything still builds because those files were indeed, useless

closes #377

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
